app/controllers/notifications_controller.rb

```diff
@@ -104,7 +104,7 @@ def index
   #   { "count" : 1 }
   #
   def unread_count
-    scope = current_user.notifications
+    scope = current_user.notifications.unmuted
```

Comment:
Let's keep the default scope the same as before, when notifications are muted, they also get archived so will disappear from the inbox already. This feature is mostly for the advanced search prefixes.

Reply:
will update